src/report/Report.tsx

```diff
@@ -57,17 +59,28 @@ export const NeoReport = ({
   deletePrepopulationReportFunction,
   theme,
 }) => {
-  const [records, setRecords] = useState(null);
+  const [records, setStateRecords] = useState<NeodashRecord[]>([]);
   const [timer, setTimer] = useState(null);
   const [status, setStatus] = useState(QueryStatus.NO_QUERY);
   const { driver } = useContext<Neo4jContextState>(Neo4jContext);
+  const { connectionModule } = useConnectionModuleContext();
   const [loadingIcon, setLoadingIcon] = React.useState(REPORT_LOADING_ICON);
+
+  const setRecords = (records) => {
+    let toSet = records;
+    if (['table'].includes(type)) {
```

Comment:
Remove this since it's not finished?
Add a todo for later?

Reply:
Yeah we can do it, but from next release we must start the migration to the new abstracted model